packages/webview_flutter/webview_flutter_platform_interface/lib/src/webview_platform.dart

```diff
@@ -10,6 +10,8 @@ import 'platform_webview_cookie_manager.dart';
 import 'platform_webview_widget.dart';
 import 'types/types.dart';
 
+// TODO(bparrishMines): This should be removed once webview_flutter_android and
+// webview_flutter_wkwebview no longer depend on this file in tests.
 export 'types/types.dart';
```

Comment:
This is an implementation file and is being used by our platform implementations tests for this export: https://github.com/flutter/packages/blob/main/packages/webview_flutter/webview_flutter_android/test/android_webview_controller_test.dart#L21

I think this export may had caused the IDE to choose this file to import when accessing the classes from this file. Instead of the `package:webview_flutter_platform_interface/webview_flutter_platform_interface.dart`. In the platform PRs I'll change the imports in the tests.

Reply:
Aren't there types that are part of this interface that come from there? Like the param object types?

Reply:
Yes, but it's included in the `webview_flutter_platform_interface.dart` file: https://github.com/flutter/packages/blob/main/packages/webview_flutter/webview_flutter_platform_interface/lib/webview_flutter_platform_interface.dart#L9